backend/FwLite/MiniLcm/SyncHelpers/DiffCollection.cs

```diff
@@ -28,12 +28,25 @@ public override Guid GetId(T value)
     }
 }
 
-public interface IOrderableCollectionDiffApi<T> where T : IOrderable
+public interface IOrderableCollectionDiffApi<T>
 {
     Task<int> Add(T value, BetweenPosition between);
     Task<int> Remove(T value);
     Task<int> Move(T value, BetweenPosition between);
     Task<int> Replace(T before, T after);
+    Guid GetId(T value);
```

Comment:
I'd provide an implementation for this method (yes you can do that on interfaces) and back the `where T : IOrderable` then in ComplexFormComponentsDiffApi you can just change `GetId`

Reply:
fixed this myself